packages/mongodb-chatbot-server/src/routes/responses/createResponse.test.ts

```diff
@@ -246,6 +246,101 @@ describe("POST /responses", () => {
 
       expect(response.statusCode).toBe(200);
     });
+
+    it("Should return 200 if conversation store flag is undefined (should default to true) and store is true", async () => {
+      const conversation =
+        await appConfig.conversationsRouterConfig.conversations.create({
+          initialMessages: [
+            {
+              role: "user",
+              content: "What is MongoDB?",
+              customData: {
+                store: undefined,
+              },
+            },
+          ],
+        });
+
+      const previousResponseId = conversation.messages[0].id.toString();
+      const response = await makeCreateResponseRequest({
+        previous_response_id: previousResponseId,
+        store: true,
+      });
+
+      expect(response.statusCode).toBe(200);
+    });
+
+    it("Should properly store conversations when store flag is true", async () => {
+      const createSpy = jest.spyOn(
```

Comment:
would it be better practice to clear the spies after every test? i'm not very familiar w/ using spies (tho seems handy)